### PR TITLE
Remove discrepancy in help message

### LINKED ISSUE #252: Discrepancy in help message
When we run coala-quickstart, at first it asks for project directory and at that time it shows no hint or help message as what is project directory. After entering (or not entering) the project directory, there appears an help message telling what we should enter for the next question (files to ignore) and what we should enter for the first question which is quite odd as the user has already entered the answer to the first question!
coala

The above mentioned behavior can be mentioned in the above screen shot. There should be a separate help message for the first question shown when the first question itself is asked.

### PR DESCRIPTION
**Pull request by djmgit**
coala_quickstart/Strings.py: Updated GLOB_HELP message and added
PROJECT_DIR_HELP message.

coala_quickstart.py: Displayed help message for project directory
question.

Closes #252

### CODE REVIEW

**Comment by ishanSrt** [comment]
you need to increase test coverage

**Comment by ishanSrt** [comment]
also check if the help messages are meaningful in case of `--non-interactive` session, and fix that over here too

**Comment by djmgit** [comment]
@ishanSrt will update my PR ASAP.

**Comment by jayvdb** [comment]
We can ignore the codecov diff error if this is simply changing some UI messages and not introducing new branches in the logic.

**Comment by djmgit** [comment]
@jayvdb @ishanSrt I have checked the ```--non-interactive``` session, the help message is not showing up, as it should be.

Also, presently help messages are not tested. For example GLOB_HELP test message is not tested. So should I go for test here?

**Comment by ishanSrt** [comment]
>For example GLOB_HELP test message is not tested. So should I go for test here?

yes QuickStart already lacks behind on coverage than other repos, if we've gotta chance to increase tests, we should never fall behind

**Comment by ishanSrt** [comment]
No that issue is opened recently and not related to this issue, add tests over here itself, for your changes

**Comment by djmgit** [comment]
@ishanSrt @jayvdb not sure why travis is failing. I have ran tests locally using ```python setup.py test``` and the tests are running without any error.

**Comment by jayvdb** [comment]
The logs (https://travis-ci.org/coala/coala-quickstart/jobs/392166113) show 

```py
Traceback (most recent call last):
  ...
  File "/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages/_pytest/vendored_packages/pluggy.py", line 515, in load_setuptools_entrypoints
    "Plugin %r could not be loaded: %s!" % (ep.name, e))
_pytest.vendored_packages.pluggy.PluginValidationError: Plugin 'timeout' could not be loaded: (pytest 3.0.7 (/home/travis/virtualenv/python3.4.6/lib/python3.4/site-packages), Requirement.parse('pytest>=3.6.0'))!
```

This was a problem fixed by https://github.com/coala/coala-quickstart/pull/255

**Comment by jayvdb** [comment]
@gitmate-bot rebase

File: tests/interaction/Logo.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 from coala_utils.ContextManagers import retrieve_stdout
 from coala_quickstart.interaction.Logo import (
     print_welcome_message, print_side_by_side)
+from coala_quickstart.Strings import (
+    GLOB_HELP, PROJECT_DIR_HELP)
```
[the remark this answers is not in the document]
Comment (ishanSrt) [reply tests/interaction/Logo.py]: https://github.com/coala/coala-quickstart/issues/256#issuecomment-397817700

**Comment by djmgit** [comment]
@jayvdb please have a look, I have done the requested changes.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-quickstart/pull/253/commits/14630a9a5ab3d06519fbad10df3a37759253d916

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**